src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/TrancheManager.java

```diff
@@ -136,61 +167,50 @@ private static void writeTranchesDebuggingInfo(final File f, final List<VariantD
         }
     }
 
-    public static Tranche findTranche(
+    public static TruthSensitivityTranche findTranche(
             final List<VariantDatum> data,
             final SelectionMetric metric,
             final double trancheThreshold,
             final VariantRecalibratorArgumentCollection.Mode model ) {
-        logger.info(String.format("  Tranche threshold %.2f => selection metric threshold %.3f", trancheThreshold, metric.getThreshold(trancheThreshold)));
+        logger.info(String.format("  TruthSensitivityTranche threshold %.2f => selection metric threshold %.3f", trancheThreshold, metric.getThreshold(trancheThreshold)));
 
         double metricThreshold = metric.getThreshold(trancheThreshold);
         int n = data.size();
         for ( int i = 0; i < n; i++ ) {
             if ( metric.getRunningMetric(i) >= metricThreshold ) {
                 // we've found the largest group of variants with sensitivity >= our target truth sensitivity
-                Tranche t = trancheOfVariants(data, i, trancheThreshold, model);
+                TruthSensitivityTranche t = TruthSensitivityTranche.trancheOfVariants(data, i, trancheThreshold, model);
                 logger.info(String.format("  Found tranche for %.3f: %.3f threshold starting with variant %d; running score is %.3f ",
                         trancheThreshold, metricThreshold, i, metric.getRunningMetric(i)));
-                logger.info(String.format("  Tranche is %s", t));
+                logger.info(String.format("  TruthSensitivityTranche is %s", t));
                 return t;
             }
         }
 
         return null;
     }
 
-    public static Tranche trancheOfVariants(
+    public static VQSLODTranche findVQSLODTranche(
             final List<VariantDatum> data,
-            final int minI,
-            final double ts,
+            final SelectionMetric metric,
+            final double trancheThreshold,
             final VariantRecalibratorArgumentCollection.Mode model ) {
-        int numKnown = 0, numNovel = 0, knownTi = 0, knownTv = 0, novelTi = 0, novelTv = 0;
-
-        double minLod = data.get(minI).lod;
-        for ( final VariantDatum datum : data ) {
-            if ( datum.lod >= minLod ) {
-                //if( ! datum.isKnown ) System.out.println(datum.pos);
-                if ( datum.isKnown ) {
-                    numKnown++;
-                    if( datum.isSNP ) {
-                        if ( datum.isTransition ) { knownTi++; } else { knownTv++; }
-                    }
-                } else {
-                    numNovel++;
-                    if( datum.isSNP ) {
-                        if ( datum.isTransition ) { novelTi++; } else { novelTv++; }
-                    }
-                }
+        logger.info(String.format("  VQSLODTranche threshold %.2f => selection metric threshold %.3f", trancheThreshold, metric.getThreshold(trancheThreshold)));
+
+        double metricThreshold = metric.getThreshold(trancheThreshold);
```

Comment:
I'm guessing you copied this from elsewhere, but while you're here you might as well add 'final's as needed.  :)